task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,13 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num % 3 === 0 && num % 5 === 0) {
+        return 'FizzBuzz'
+    } else if (num % 3 === 0) {
+        return 'Fizz'
+    } else if (num % 5 === 0) {
+        return 'Buzz'
+    } else return num
```

Comment:
We don't need to use `else if` statements if we are using early returns. Also, if we want to check if number is divisible both by 3 and 5, we can just check if it is divisible by 15.

```suggestion
    if (num % 15) {
        return 'FizzBuzz'
    } 
    
    if (num % 3 === 0) {
        return 'Fizz'
    } 
    
    if (num % 5 === 0) {
        return 'Buzz'
    } 
    
    return num
```